ext/standard/string.c

```diff
@@ -4566,14 +4566,6 @@ PHP_FUNCTION(hebrev)
 }
 /* }}} */
 
-/* {{{ proto string hebrevc(string str [, int max_chars_per_line])
-   Converts logical Hebrew text to visual text with newline conversion */
-PHP_FUNCTION(hebrevc)
-{
-	php_hebrev(INTERNAL_FUNCTION_PARAM_PASSTHRU, 1);
```

Comment:
It seems to me that we now can inline `php_hebrev()` into `PHP_FUNCTION(hebrev)`.

Reply:
Good idea! Just commited it.